### ISSUE TITLE
Application crashes when we touch the user's avatar

### ISSUE THREAD

**Issue by robson-alcantara** [NearbyShops/Nearby-Shops-Android-app#29]
Application crashes when we touch the user's avatar (compiled from source version)

**Comment by SumeetMoray** [comment]
yes ... i believe i have fixed that issue in recent commit ! ...
So try once more to build from the latest source  and check if that happens !
